cmds/dhclient/dhclient.go

```diff
@@ -76,166 +75,57 @@ func ifup(ifname string) (netlink.Link, error) {
 	return nil, fmt.Errorf("Link %v still down after %d seconds", ifname, linkUpAttempt)
 }
 
-func dhclient4(iface netlink.Link, numRenewals int, timeout time.Duration, retry int) error {
-	mac := iface.Attrs().HardwareAddr
-	conn, err := dhcp4client.NewPacketSock(iface.Attrs().Index)
+func DHClient4(iface netlink.Link, timeout time.Duration, retry int, numRenewals int) error {
```

Comment:
nit (i.e. feel free to ignore it): this and other exported functions below will need some comments. It's OK to do this in a future PR though

Reply:
fixed.